packages/theme/src/theme/_colors.scss

```diff
@@ -153,6 +153,20 @@ $lochmara800: #022e4d;
 $lochmara900: #011726;
 $lochmara: $lochmara500;
 
+/// Secondary color.
```

Comment:
```suggestion
/// Palecyan
```

Reply:
i applied the same schema as the others, don't know if it's normal or not :/ 

![image](https://user-images.githubusercontent.com/26482371/128037428-44f109d6-8ff1-4ab6-82a7-bdd0cd0cf19d.png)